### PR TITLE
Cpu

### PR DESCRIPTION
**Pull request by jacobyeung**
FIxing cpu

### CODE REVIEW

**Review by Corleno — APPROVED** [review]
That sounds good to me.

**Review by Corleno — APPROVED** [review]
I am thinking that if we use if as a module, we may not need it. This is for illustration in a real example. But at this moment, I approve it and we may need to optimize it later.

File: vbmi/mi_estimator.py [hunk 1]
```diff
@@ -10,7 +10,8 @@
 
 #Snippet of code written to take benefits of GPU computation
 train_on_GPU = cuda.is_available()
-assert train_on_GPU, "please run this code on GPU (Google Collab is free and sufficient for instance)"
+print(f'training on GPU: {train_on_GPU}')
+# assert train_on_GPU, "please run this code on GPU (Google Collab is free and sufficient for instance)"
```
Comment (jihyunbak) [inline vbmi/mi_estimator.py:14]: @Corleno - so we could remove this?

Comment (Corleno) [reply vbmi/mi_estimator.py:14]: Yes. I think so.

**Pull request merged**